cmd/eksctl-anywhere/cmd/downloadimages.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+*/
+package cmd
+
+import (
+	"context"
+	"log"
+
+	"github.com/spf13/cobra"
+
+	"github.com/aws/eks-anywhere/cmd/eksctl-anywhere/cmd/internal/commands"
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/docker"
+	"github.com/aws/eks-anywhere/pkg/executables"
+	"github.com/aws/eks-anywhere/pkg/version"
+)
+
+// imagesCmd represents the images command
+var downloadImagesCmd = &cobra.Command{
+	Use:   "images",
+	Short: "Download all eks-a images to disk",
+	Long: `Creates a tarball containing all necessary images
+to create an eks-a cluster for any of the supported
+Kubernetes versions.`,
+	SilenceUsage: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ctx := context.Background()
+
+		return downloadImagesRunner.Run(ctx)
+	},
+}
+
+func init() {
+	downloadCmd.AddCommand(downloadImagesCmd)
+
+	downloadImagesCmd.Flags().StringVarP(&downloadImagesRunner.outputFile, "output", "o", "", "Output tarball containing all downloaded images")
```

Comment:
can we include a bundles-override or similar option to download specific images? this is mostly for dev testing

Reply:
for snow case, the images downloaded and built in the admin AMI can be different from `import images` user runs in admin AMI